### ISSUE TITLE
Deployment of my project

### ISSUE THREAD

**Issue by Wils-FOURR-speed** [Wils-FOURR-speed/My-portfolio#6]
https://wils-fourr-speed.github.io/My-portfolio/

**Comment by Efiamotu-1** [comment]
@Wils-FOURR-speed Good job deploying your project on github. Your project can  now be accessed online by anyone through your link :+1: :100: :1st_place_medal: 
- [ ] However you need to add this your deployed link to your readme file in the `live demo` section so that it can easily be accessed by anyone who visits your github :+1: :sparkles: 
- Once this is done resubmit your project and it should be approved :heavy_check_mark:

**Comment by LionRouge1** [comment]
Hi @Efiamotu-1, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Suggested changes :recycle: 
- [ ] As requested by the previous reviewer, Kindly add your live project link to your README file in the `live demo` section so that it can easily be accessed by anyone who visits your GitHub :+1:
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Wils-FOURR-speed** [comment]
Please where exactly because i added it already. The README file has the
link included already.
